ui/app/src/views/ViewDashboard.tsx

```diff
@@ -31,6 +31,23 @@ import { useDatasourceApi } from '../model/datasource-api';
 // https://apache.github.io/echarts-handbook/en/concepts/style/#theme
 const ECHARTS_THEME_OVERRIDES = {};
 
+interface DashboardBreadcrumbsProps {
+  dashboardName: string;
+  dashboardProject: string;
+}
+
+function DashboardBreadcrumbs(props: DashboardBreadcrumbsProps) {
+  const { dashboardName, dashboardProject } = props;
+  return (
+    <Breadcrumbs sx={{ fontSize: 'large' }}>
+      <Link underline={'hover'} variant={'h2'} component={RouterLink} to={`/projects/${dashboardProject}`}>
+        {dashboardProject}
+      </Link>
+      <Typography variant={'h2'}>{dashboardName}</Typography>
```

Comment:
Should this actually be the H1? Currently, we have 'Perses' in the future logo spot as the H1, which isn't really correct ([ex](https://webmasters.stackexchange.com/a/4523)) since it isn't unique / descriptive about the current page like using the dashboardName here would be.